github/ci/prow/files/config.yaml

```diff
@@ -113,6 +115,39 @@ branch-protection:
           branches:
             master:
               protect: true
+        kubevirtci:
```

Comment:
@rmohr @fabiand this is the best compromise I can come up with currently. Everytime I try to protect the whole org the jobs runs into a timeout for reasons I don't understand. AFAIU this should be good enough for now.

Makes sense?